src/leappto/cli.py

```diff
@@ -169,10 +169,10 @@ def _path_spec(arg):
     _add_identity_options(migrate_cmd, context='source')
     _add_identity_options(migrate_cmd, context='target')
 
-    check_target_cmd.add_argument('target', help='target VM name')
+    check_target_cmd.add_argument('-t', '--target', default='localhost', help='Target running docker')
```

Comment:
I'd suggest "Target container host" as the option description (while we currently require docker specifically, I assume we'll eventually support any OCI compliant container management daemon)